osu.Game.Rulesets.Osu/Mods/OsuModDifficultyAdjust.cs

```diff
@@ -30,6 +31,22 @@ public class OsuModDifficultyAdjust : ModDifficultyAdjust
             Value = 5,
         };
 
+        public override string SettingDescription
+        {
+            get
+            {
+                string circleSize = CircleSize.IsDefault ? "" : $"CS {CircleSize.Value}";
+                string drainRate = DrainRate.IsDefault ? "" : $"HP {DrainRate.Value}";
+                string overallDifficulty = OverallDifficulty.IsDefault ? "" : $"OD {OverallDifficulty.Value}";
+                string approachRate = ApproachRate.IsDefault ? "" : $"AR {ApproachRate.Value}";
+
+                string[] settings = { circleSize, drainRate, overallDifficulty, approachRate };
```

Comment:
repeated implementation for `drainRate` / `overallDifficulty`? why not use `base.` for those?

Reply:
no reason other than I forgot to. fixed